copy-builtin

```diff
@@ -41,8 +41,8 @@ scripts/make_gitrev.sh || true
 rm -rf "$KERNEL_DIR/include/zfs" "$KERNEL_DIR/fs/zfs"
 cp --recursive include "$KERNEL_DIR/include/zfs"
 cp --recursive module "$KERNEL_DIR/fs/zfs"
-mkdir "$KERNEL_DIR/fs/contrib"
-cp --recursive contrib/zstd "$KERNEL_DIR/fs/contrib"
+mkdir "$KERNEL_DIR/fs/zfs/contrib"
+cp --recursive contrib/zstd "$KERNEL_DIR/fs/zfs/contrib"
```

Comment:
@BrainSlayer What if the build-in fix is the issue here?
Duplicate building paths might in theorie lead to double execution....

Reply:
no, we had the same issues without